CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-NONE**
+  - Adds logging to ecs/async-operation Docker conatiner that launches async
```

Comment:
I'm not sure this is right, the updates are to the async-operations repo. 

maybe 'Updates default async operation image version to 39 to take advantage of (link to changelog) logging updates'?

Reply:
Wihdrawn.    😓